.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "schemas"]
 	path = schemas
 	url = https://github.com/open-feature/schemas
+	branch = main
```

Comment:
This is safe because we aren't using `git submodule update --remote` in any automation, but if we did, it would introduce some nondeterminism into the build since every time such a command would run it would get the current head of `main`, instead of a specific SHA:

https://git-scm.com/docs/git-submodule#Documentation/git-submodule.txt---remote

It is nice for making it obvious to people how to reference a different branch of the submodule for local work, so I think it makes sense to keep it.